mmv1/templates/terraform/custom_import/vertex_ai_featurestore_entitytype_feature.go.erb

```diff
@@ -0,0 +1,22 @@
+config := meta.(*Config)
+if err := parseImportId([]string{
+	"(?P<entitytype>.+)/features/(?P<name>[^/]+)",
+}, d, config); err != nil {
+	return nil, err
+}
+
+// Replace import id for the resource id
+id, err := replaceVars(d, config, "{{entitytype}}/features/{{name}}")
+if err != nil {
+	return nil, fmt.Errorf("Error constructing id: %s", err)
+}
+d.SetId(id)
+
+entitytype := d.Get("entitytype").(string)
+
+re := regexp.MustCompile("^projects/(.+)/locations/(.+)/featurestores/(.+)$")
```

Comment:
Would we want to validate that the entity type also includes `entityTypes/`, or am I misunderstanding this logic?

Reply:
The purpose of the regex is just to extract the `<location>` from `projects/<project name>/locations/<location>/featurestores/<featurestore name>/entityTypes/<entitytype name>`. 

I didn't intend to validate the id of entity type here, but adding the validation sounds good to me. So I'll update the regex. Thanks!

Reply:
I've updated it in 047a83b.